src/Console/DacapoReverseEngineerCommand.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace UcanLab\LaravelDacapo\Console;
+
+use Illuminate\Console\Command;
+use Illuminate\Console\ConfirmableTrait;
+use UcanLab\LaravelDacapo\Storage\ModelsStorage;
+use UcanLab\LaravelDacapo\Storage\SchemasStorage;
+use UcanLab\LaravelDacapo\Migrations\SchemaLoader;
+use UcanLab\LaravelDacapo\Generator\ModelTemplateGenerator;
+
+/**
+ * Class DacapoModelsCommand.
+ */
+class DacapoReverseEngineerCommand extends Command
+{
+    use ConfirmableTrait;
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'dacapo:reverse-engineer
+    ';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Generate template models.';
+
+    /**
+     * @return void
+     */
+    public function handle(): void
+    {
+        $tables = collect(\DB::select('SHOW TABLES'))->map(function (\StdClass $table) : string {
+            return collect($table)->values()->first();;
```

Comment:
__[PHP_CodeSniffer]__ Each PHP statement must be on a line by itself ([link](https://sider.review/gh/repos/151422991/pulls/70#issue-4594105))

<sub>posted by Sider</sub>